client/js/components/MapView/MapView.js

```diff
@@ -1,9 +1,29 @@
 import React, { Component } from "react";
-import { TouchableOpacity } from "react-native";
+import { TouchableOpacity, Text } from "react-native";
 import MapView, { PROVIDER_GOOGLE } from "react-native-maps";
 import styles from "./styles";
+import IconFontAwesome from "react-native-vector-icons/FontAwesome";
```

Comment:
Did you make a pull request for installing/linking this library?

Reply:
Yes I did... a long time ago